modules/data.atmosphere/R/met2model.module.R

```diff
@@ -41,7 +41,7 @@
     fcn <- paste0("met2model.", model)
     lst <- site.lst(site.id=site$id, con=con)
     
-    model.id <- PEcAn.utils::convert.input(input.id = input.id,
+    model.id <- convert.input(input.id = input.id,  ###PEcAn.utils::
```

Comment:
Old version was correct: Please use `pkgname::function()` consistently for all functions that live outside the current package.

Reply:
Whoops, that's something I did for debugging.  I'll definitely change that back.  I can't believe I missed that.